modules/network-graph-20-view.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_module-type: CONCEPT
+[id="network-graph-20-view_{context}"]
+= About the network graph (2.0 preview)
+
+[role="_abstract"]
+
+The network graph provides high-level and detailed information about deployments, network flows, and network policies in your environment.
+
+:FeatureName: Network graph 2.0
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+{product-title-short} processes all network policies in each secured cluster to show you which deployments can contact each other and which can reach external networks. It also monitors running deployments and tracks traffic between them. You can view the following items in the network graph:
+
+* Network components: From the top menu, you can select namespaces (indicated by the *NS* label) and deployments (indicated by the *D* label) to display on the graph for a chosen cluster (indicated by the *CL* label). You can further filter deployments by using the drop-down list and selecting criteria on which to filter such as common vulnerabilities and exposures (CVEs), labels, and images.
+* External entities: These represent entities that are connected outside of your cluster. For more information, see "External entities and connections."
+* Network policies: You can view existing policies for a selected component or view components that have no policies.
+* Network flows: You can select one of the following flows for the graph:
+** Active traffic (default): Selecting this option shows shows observed traffic, focused on the namespace or specific deployment that you selected. You can select the time period for which to display information.
+** Inactive flows: Selecting this option shows potential flows allowed by your network policies, helping you identify missing network policies needed to achieve tighter isolation. You can select the time period for which to display information.
+
+You can also simulate network policies from the network graph view. See "Simulating network policies" for more information.
+
+[discrete]
+== Navigation and user interface in Network Graph (2.0 preview)
+
+* Clicking on items or connections between items in the graph allows you to view additional information about components or connections, or perform actions such as adding a network flow to your baseline. You can also click on the connections between items to view information about the connection.
```

Comment:
@kcarmichael08 @vjwilson I don't think we allow clicking on connections. Would this need to be changed?

Reply:
Will remove for 2.0 version